reflex/components/el/elements/__init__.pyi

```diff
@@ -338,5 +338,5 @@ _MAPPING = {
     ],
 }
 EXCLUDE = ["del_", "Del", "image"]
-for _, v in _MAPPING.items():
+for (_, v) in _MAPPING.items():
```

Comment:
I don't think this change should be here.

It's making the pyi check in the CI fail.

Reply:
Hi @Lendemor here is what I have in my `reflex/components/el/elements/__init__.pyi` file.
```
EXCLUDE = ["del_", "Del", "image"]
for (_, v) in _MAPPING.items():
    v.extend([mod.capitalize() for mod in v if mod not in EXCLUDE])
```
If I understand correctly, these are generated for the component

Reply:
For component they are generated by running `python scripts/make_pyi reflex/components/<path/to/file>`.

It'll be run automatically if you use `pre-commit`

Reply:
Okay that makes sense. 

I skipped pre-commit with `--no-verify` since my Python version is below 3.12.

But I ran `poetry run python scripts/make_pyi.py ` as stated in the `CONTRIBUTING.md` guide. I guess since I omitted 
calling the path to the component file `reflex/components/<path/to/file>` then I had the issue.

Thanks for your help. They were super useful